test/hexpm_web/controllers/package_controller_test.exs

```diff
@@ -130,6 +130,19 @@ defmodule HexpmWeb.PackageControllerTest do
       assert response(conn, 200) =~ escape(~s({:#{package1.name}, "~> 0.0.1"}))
     end
 
+    test "show publisher info", %{package1: package1} do
+      insert(
+        :release,
+        package: package1,
+        publisher: build(:user, username: "Publisher"),
+        version: "0.1.0",
+        meta: build(:release_metadata, app: package1.name)
+      )
+
+      conn = get(build_conn(), "/packages/#{package1.name}/0.1.0")
+      assert response(conn, 200) =~ "Publisher"
```

Comment:
"Publisher" is the name of the heading, so to avoid false positives let's test for publisher's username; e.g.:

```elixir
assert response(conn, 200) =~ release.publisher.username
```

We can leave publisher username to be random.

Reply:
Yes, you are absolutely right! Fixed in 41a5939